### PR TITLE
Add Readme.md to project

### PR DESCRIPTION
**Pull request by CodrinChirica**
- add notes for user to state that the version in incubator is subject to change

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Alfresco Kubernetes Charts
+
+Source repository for *Alfresco Kubernetes Charts*
+
+## Disclaimer 
+
+As an end user of the charts, this is to inform you that for latest stable released version, use the charts in *stable* directory. 
+The charts in *incubator* are subject to change as we are overriding the version with every new release.
```
Comment (cristina-sirbu) [inline README.md]: I think I would change the order of the sentences to emphasize the warning about incubator, or some other way to get peoples attention on that.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Alfresco Kubernetes Charts
+
+Source repository for *Alfresco Kubernetes Charts*
```
Comment (rgauss) [inline README.md]: This isn't actually the source, maybe:
> Helm repo for ...

Comment (rgauss) [reply README.md]: Let's also repeat what's in the repo description in this section, except the source part 😄

File: README.md (hunk 1 above)
Comment (rgauss) [inline README.md]: Maybe "... as we may override the version with new releases."

**Review by rgauss — CHANGES_REQUESTED** [review]

**Review by cristina-sirbu — APPROVED** [review]

**Review by rgauss — APPROVED** [review]

**Pull request merged**